examples/nonautoregressive_translation/README.md

```diff
@@ -110,7 +110,7 @@ fairseq-generate \
     data-bin/wmt14_en_de_distill \
     --gen-subset test \
     --task translation_lev \
-    --path checkpoints/checkpoint_best.pt:at_checkpoints/checkpoint_best.pt \
+    --path checkpoints/checkpoint_best.pt::at_checkpoints/checkpoint_best.pt \
```

Comment:
Let's revert back to `:` and use `os.pathsep` when parsing.

Reply:
@myleott done!